packages/editor/src/components/DiagramPanel.tsx

```diff
@@ -177,9 +210,16 @@ export default function DiagramPanel() {
       }
       // TODO: publish images in the gist
       case "gist":
-      // TODO: cache images in local storage?
-      case "local":
         return undefined;
+      case "local": {
+        return fetchUrl(
+          relativePath,
+          new URL(
+            relativePath,
+            window.location.origin + window.location.pathname + "svg/"
```

Comment:
Instead of hard-coding the `svg/`  path, `local` workspace should only get the image from local storage because the image must came from gist, example, roger, or file upload (this is not implemented but doesn't change the logic)

Reply:
Removed.  This breaks the ability for example images to work for trios that are pasted in, but that doesn't sound like a user case we are supporting right now, which would make this unnecessary.